tsconfig.json

```diff
@@ -1,5 +1,9 @@
 {
     "extends": "@box/frontend/ts/tsconfig.json",
-    "include": ["./src/**/*.ts", "./src/**/*.tsx"],
-    "exclude": ["node_modules", "dist"]
+    "compilerOptions": {
```

Comment:
Can you try upgrading `@box/frontend`, first? I think this was fixed in the latest version.